.ci/bump-version.sh

```diff
@@ -11,12 +11,13 @@ docker run --rm -t \
   -w /app \
   -v "$(pwd):/app" \
   node:12-alpine /bin/sh -c "set -x
-    CI=true npm install elastic-apm-node@${AGENT_VERSION}"
+    CI=true npm install --ignore-scripts elastic-apm-node@${AGENT_VERSION}"
```

Comment:
Just noting we've added  `--ignore-scripts` here.  I'm presuming this is related to discussion around SLSA/Supply Chain mitigation.

Reply:
Partially related to the SLSA discussions, but also to avoid this current package.json "scripts" entry:

```
    "postinstall": "if [ \"$NODE_ENV\" != production ]; then npm run client-update; fi",
```

which I think may be adding a lot of unnecessary time to this "bump-version.sh" run.